system/doc/design_principles/statem.xml

```diff
@@ -1693,12 +1693,12 @@ open(cast, {button,_}, Data) ->
     <p>
       If you need to cancel a timer because of some other event, you can use
       <seealso marker="erts:erlang#cancel_timer/2"><c>erlang:cancel_timer(Tref)</c></seealso>.
-      Note that a time-out message cannot arrive after this,
-      unless you have postponed it before (see the next section),
+      Note that no time-out message will arrive after this (because the timer has been explicitly canceled),
+      unless you have already postponed one earlier (see the next section),
       so ensure that you do not accidentally postpone such messages.
-      Also note that a time-out message may have arrived
-      just before you cancelling it, so you may have to read out
-      such a message from the process mailbox depending on
+      Also note that a time-out message may arrive during a **state callback** 
```

Comment:
To be more precise I suggest change to "...during **the** state callback...".

Also "\*\*state callback\*\*" should be "\<em\>state callback\</em\>" in our documentation .xml files.